iosHackthonApp/Modules/Features/MealList/Sources/MealList/MealListViewModel.swift

```diff
@@ -40,7 +41,7 @@ public final class MealListViewModel: ObservableObject {
             guard
                 let meals = meals,
                 error == nil else {
-                assertionFailure(error!.localizedDescription)
+                print(error!.localizedDescription)
```

Comment:
Hmm does this mean we should maybe be showing some sort of error state to the user?

Reply:
Yeah, we should show an alert or something. I added this just as a catch to remind me that I didn't start the `json-server` yet 😬 but I think that assertion was causing my CI problems last night/this morning.